<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDs4</class>
 <widget class="QWidget" name="IDs4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gen 4 TID/SID</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxSearcher">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Searcher</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBoxFilters">
        <property name="title">
         <string>Filters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="IDsFilter" name="idFilter" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBoxRNGInfo">
        <property name="title">
         <string>RNG Info</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="1">
          <widget class="TextBox" name="textBoxYear">
           <property name="text">
            <string notr="true">2000</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBoxInfiniteSearch">
           <property name="text">
            <string>Infinite Search</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButtonSearch">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButtonCancel">
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <widget class="TextBox" name="textBoxSearcherMaxDelay">
           <property name="text">
            <string notr="true">6000</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelSearcherMaxDelay">
           <property name="text">
            <string>Max Delay</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="TextBox" name="textBoxSearcherMinDelay">
           <property name="text">
            <string notr="true">5000</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelSearcherMinDelay">
           <property name="text">
            <string>Min Delay</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelYear">
           <property name="text">
            <string>Year</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QProgressBar" name="progressBar"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxSeedFinder">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Seed Finder</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTID">
        <property name="text">
         <string>TID Obtained</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="TextBox" name="textBoxTID"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelDateTime">
        <property name="text">
         <string>Date/Time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelSeedFinderMinDelay">
        <property name="text">
         <string>Min Delay</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="TextBox" name="textBoxSeedFinderMinDelay">
        <property name="text">
         <string notr="true">5000</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelSeedFinderMaxDelay">
        <property name="text">
         <string>Max Delay</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="TextBox" name="textBoxSeedFinderMaxDelay">
        <property name="text">
         <string notr="true">6000</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="pushButtonFind">
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="TableView" name="tableViewSearcher"/>
   </item>
   <item row="2" column="1">
    <widget class="TableView" name="tableViewSeedFinder"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Form/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>IDsFilter</class>
   <extends>QWidget</extends>
   <header>Form/Controls/IDsFilter.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Form/Controls/TableView.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>textBoxYear</tabstop>
  <tabstop>textBoxSearcherMinDelay</tabstop>
  <tabstop>textBoxSearcherMaxDelay</tabstop>
  <tabstop>checkBoxInfiniteSearch</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>tableViewSearcher</tabstop>
  <tabstop>textBoxTID</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>textBoxSeedFinderMinDelay</tabstop>
  <tabstop>textBoxSeedFinderMaxDelay</tabstop>
  <tabstop>pushButtonFind</tabstop>
  <tabstop>tableViewSeedFinder</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
